### PR TITLE
Add support for retrieving a single notification info using his ID

### LINKED ISSUE #488: [Notification] Get notification by id
I want to get a single notification info by using notification ID.
Github DOCS: https://developer.github.com/v3/activity/notifications/#view-a-single-thread
(Using Laravel-Github)
Thank you!

### PR DESCRIPTION
**Pull request by m1guelpf**
Fix #488
Add support for retrieving a single notification info using his ID.

### CODE REVIEW

File: lib/Github/Api/Notification.php [hunk 1]
```diff
@@ -57,4 +57,19 @@ public function markRead(DateTime $since = null)
...
         $this->put('/notifications', $parameters);
     }
+    /**
+     * [UNDOCUMENTED] Gets a single notification using his ID
+     *
+     * @link https://developer.github.com/v3/activity/notifications/#view-a-single-thread
+     *
+     * @param ID|integer $id
+     */
+    public function id(integer $int)
```
Comment (m1guelpf) [inline lib/Github/Api/Notification.php]: I am having trouble with the ```ìntegrer``` imput type. It should be a number. What sould I put here?

File: lib/Github/Api/Notification.php [hunk 2]
```diff
@@ -57,4 +57,19 @@ public function markRead(DateTime $since = null)
...
+     * [UNDOCUMENTED] Gets a single notification using his ID
+     *
+     * @link https://developer.github.com/v3/activity/notifications/#view-a-single-thread
+     *
+     * @param ID|integer $id
+     */
+    public function id($id)
+    {
+        if (!is_numeric($id)) {
+            // Error
```
Comment (m1guelpf) [inline lib/Github/Api/Notification.php]: What do I do here?

File: lib/Github/Api/Notification.php [hunk 3]
```diff
@@ -57,4 +57,19 @@ public function markRead(DateTime $since = null)
 
         $this->put('/notifications', $parameters);
     }
+    /**
+     * [UNDOCUMENTED] Gets a single notification using his ID
```
Comment (Nyholm) [inline lib/Github/Api/Notification.php]: Why "UNDOCUMENTED"?

File: lib/Github/Api/Notification.php [hunk 4]
```diff
@@ -57,4 +57,19 @@ public function markRead(DateTime $since = null)
...
+     *
+     * @link https://developer.github.com/v3/activity/notifications/#view-a-single-thread
+     *
+     * @param ID|integer $id
```
Comment (Nyholm) [inline lib/Github/Api/Notification.php]: Should be `@param int $id`

File: lib/Github/Api/Notification.php (hunk 2 above)
Comment (Nyholm) [inline lib/Github/Api/Notification.php]: It's okey if you remove this if statement. If someone uses a string they will get a 404 back.

File: lib/Github/Api/Notification.php (hunk 3 above)
Comment (m1guelpf) [inline lib/Github/Api/Notification.php]: It isn't in this library documentation

Comment (Nyholm) [inline lib/Github/Api/Notification.php]: I suggest to remove "[UNDOCUMENTED]" and add some documentation instead =)

**Comment by m1guelpf** [comment]
@Nyholm Done

**Comment by m1guelpf** [comment]
@Nyholm Merge?

**Comment by Nyholm** [comment]
I do not have push permission to this repo. (I would be happy if I was granted that though) 

We have to wait until @cursedcoder reviews this. Im sure he will not find any issues.

**Comment by cursedcoder** [comment]
@Nyholm just sent you an invitation

**Comment by Nyholm** [comment]
Thank you @cursedcoder 

And thank you @m1guelpf for the PR.

**Pull request merged**